client/app/styles/app.scss

```diff
@@ -172,3 +172,13 @@ $sticky-sidebar-offset: 6rem + rem-calc(20);
 .dropdown-clear-btn-below {
   color: $red-dark;
 }
+
+// Styling for tooltip "i" icons
+// note: over the top specifity was required to overwrite the fa defaults
+span.ember-tooltip-target > svg.svg-inline--fa.fa-info-circle.fa-w-16.ember-view.ember-tooltip-target {
```

Comment:
Interesting. I would almost prefer using `!important` instead of child selector. 
I can't speak to performance of one vs the other, but `!important` seems more absolute  and less flaky.

Reply:
Also I would prefer creating a class like "radio-tooltips" or something

Reply:
I avoid using `!important` as bad practice, reference here:  [MDN docs on specificity: !important](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#:~:text=Although%20technically%20!important%20has%20nothing,natural%20cascading%20in%20your%20stylesheets.). I will be using the tooltip in different `.hbd` files in the applicant portal. 

As for the classes used, I selected the one rendered by the `EmberTooltip` component  to achieve the specificity needed to override the default styles.

Reply:
Sounds good. Thanks for sharing that. If you haven't already. I would also suggest looking into using the FaIcon `@ size` attribute first before sizing it in css 
```
@size="4x"
```
your call

Reply:
I tried the size attribute previously but I couldn't get it smaller than 1em/16px (used `@size="1x"`)